### PR TITLE
Added random "bad guy with AKs" loadout.

### PR DESCRIPTION
**Pull request by Jaegerhaus**

### CODE REVIEW

File: Premade loadouts/GUE_Random_AKs.sqf [hunk 1]
```diff
@@ -0,0 +1,162 @@
+this = _this;
```
Comment (jameslkingsley) [inline Premade loadouts/GUE_Random_AKs.sqf]: See #2

File: Premade loadouts/GUE_Random_AKs.sqf [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+removeAllWeapons this;
+removeAllItems this;
+removeAllAssignedItems this;
+removeUniform this;
+removeVest this;
+removeBackpack this;
+removeHeadgear this;
+removeGoggles this;
+
+this forceAddUniform (_uniforms select floor random count _uniforms);
```
Comment (jameslkingsley) [inline Premade loadouts/GUE_Random_AKs.sqf]: `_uniforms select floor random count _uniforms` could be replaced with the faster and simpler `selectRandom _uniforms`

File: Premade loadouts/GUE_Random_AKs.sqf [hunk 3]
```diff
@@ -0,0 +1,162 @@
+this = _this;
+
+_uniforms = [
```
Comment (jameslkingsley) [inline Premade loadouts/GUE_Random_AKs.sqf]: Should add `private` to these vars to protect its scope as these files are included into the assign gear function.

Comment (Jaegerhaus) [inline Premade loadouts/GUE_Random_AKs.sqf]: Yea good catch. Was wondering where `_typeOfUnit` came from.

**Pull request merged**